services/consensusalgo/leanhelixconsensus/block_provider.go

```diff
@@ -189,3 +189,33 @@ func validLeanHelixBlockPair(blockPair *protocol.BlockPairContainer) error {
 func (p *blockProvider) GenerateGenesisBlockProposal(ctx context.Context) (lh.Block, lhprimitives.BlockHash) {
 	return nil, nil
 }
+
+func (s *Service) verifyChainTip(ctx context.Context, blockPair *protocol.BlockPairContainer, prevBlockPair *protocol.BlockPairContainer) error {
```

Comment:
i couldn't see anyone else using the other funciton so maybe merge these two funcs ?